ush/forecast_postdet.sh

```diff
@@ -133,7 +134,12 @@ FV3_postdet() {
         IAU_DELTHRS=0
         IAU_INC_FILES="''"
       fi
-
+      if [[ ${DO_LAND_IAU} = ".true." ]]; then
+        DO_LAND_IAU=".false."
+        LAND_IAU_FHRS=-1
+        LAND_IAU_DELHRS=0
+        LAND_IAU_INC_FILES="''"
+      fi
```

Comment:
I don't think this if condition is necessary.  
These are the defaults when doing a RERUN
```suggestion
      DO_LAND_IAU=".false."
      LAND_IAU_FHRS=-1
      LAND_IAU_DELHRS=0
      LAND_IAU_INC_FILES="''"
```